x-pack/plugins/uptime/public/components/monitor_management/monitor_list/monitor_list.tsx

```diff
@@ -162,8 +199,9 @@ export const MonitorManagementList = ({
         itemId="monitor_id"
         items={monitors}
         columns={columns}
-        tableLayout={'auto'}
+        tableLayout={'fixed'}
```

Comment:
Any reason why we are changing this from auto to fixed? this makes the columns awkward, leaving less space for URl column

Reply:
Thanks for pointing out. I've fixed the behavior.